policy/modules/services/podman.te

```diff
@@ -39,6 +39,12 @@ userdom_user_application_domain(podman_user_conmon_t, conmon_exec_t)
 
 allow podman_t podman_conmon_t:process setsched;
 
+kernel_rw_vm_overcommit_sysctl(podman_t)
+
+init_use_fds(podman_t)
+init_setattr_stream_sockets(podman_t)
+init_write_runtime_socket(podman_t)
```

Comment:
Seems like it would be related to `connectto`/`sendto`.  It would be best to use that interface instead.